bitwarden_license/bit-web/src/app/secrets-manager/overview/overview.component.ts

```diff
@@ -191,12 +226,57 @@ export class OverviewComponent implements OnInit, OnDestroy {
         switchMap(() => this.view$.pipe(take(1))),
         takeUntil(this.destroy$),
       )
-      .subscribe((view) => {
-        this.showOnboarding = Object.values(view.tasks).includes(false);
-        this.loading = false;
+      .subscribe({
+        next: async (view) => {
+          this.showOnboarding = Object.values(view.tasks).includes(false);
+          this.loading = false;
+        },
       });
   }
 
+  private locateOrganizationsWithIncomingPaymentIssues(
+    organization: Organization,
+    sub: OrganizationSubscriptionResponse,
```

Comment:
How about `organizationSubscriptionResponse` or `subscriptionResponse` or `subscription`
```suggestion
    organizationSubscription: OrganizationSubscriptionResponse,
```